Can not run vep

I can run normal vep tool with the same input file but errors occured when I run cannoli vep with following command: 
cannoli-submit --master local --deploy-mode client -- vep /input/sample.vcf /output/out -executable /home/spark/ensembl-vep/vep -cache file:///home/spark/.vep/homo_sapiens/95_GRCh38/ -single
Log:
`Caused by: htsjdk.tribble.TribbleException$InvalidHeader: Your input file has a malformed header: We never saw the required CHROM header line (starting with one #) for the input VCF file
	at htsjdk.variant.vcf.VCFCodec.readActualHeader(VCFCodec.java:119)
	at org.bdgenomics.adam.rdd.variant.VCFOutFormatter.read(VCFOutFormatter.scala:104)
	at org.bdgenomics.adam.rdd.OutFormatterRunner.<init>(OutFormatter.scala:32)
	at org.bdgenomics.adam.rdd.GenomicDataset$$anonfun$15.apply(GenomicDataset.scala:876)
	at org.bdgenomics.adam.rdd.GenomicDataset$$anonfun$15.apply(GenomicDataset.scala:838)
	at org.apache.spark.rdd.RDD$$anonfun$mapPartitions$1$$anonfun$apply$23.apply(RDD.scala:801)
	at org.apache.spark.rdd.RDD$$anonfun$mapPartitions$1$$anonfun$apply$23.apply(RDD.scala:801)
	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:324)
	at org.apache.spark.rdd.RDD.iterator(RDD.scala:288)
	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:324)
	at org.apache.spark.rdd.RDD.iterator(RDD.scala:288)
	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:324)
	at org.apache.spark.rdd.RDD.iterator(RDD.scala:288)
	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:324)
	at org.apache.spark.rdd.RDD.iterator(RDD.scala:288)
	at org.apache.spark.rdd.MapPartitionsRDD.compute(MapPartitionsRDD.scala:52)
	at org.apache.spark.rdd.RDD.computeOrReadCheckpoint(RDD.scala:324)
	at org.apache.spark.rdd.RDD.iterator(RDD.scala:288)
	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:90)
	at org.apache.spark.scheduler.Task.run(Task.scala:121)
	at org.apache.spark.executor.Executor$TaskRunner$$anonfun$10.apply(Executor.scala:402)
	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:1360)
	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:408)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)`
Please help to resolve the issue

--- a/core/src/main/scala/org/bdgenomics/cannoli/Vep.scala
+++ b/core/src/main/scala/org/bdgenomics/cannoli/Vep.scala
@@ -62,6 +62,9 @@ class VepArgs extends Args4jBase {
 
   @Args4jOption(required = true, name = "-cache", usage = "Ensembl VEP cache directory to use.")
   var cachePath: String = null
+
+  @Args4jOption(required = false, name = "-vep_args", usage = "Additional arguments for Ensembl VEP, must be double-quoted, e.g. -vep_args \"--af --distance 2000\"")
+  var vepArgs: String = null
 }
 
 /**
@@ -85,10 +88,9 @@ class Vep(
       .add("vcf")
       .add("--output_file")
       .add("STDOUT")
+      .add("--vcf")
       .add("--vcf_info_field")
       .add("ANN")
-      .add("--terms")
-      .add("so")
       .add("--no_stats")
       .add("--offline")
       .add("--dir_cache")
@@ -96,6 +98,7 @@ class Vep(
 
     Option(args.species).foreach(builder.add("--species").add(_))
     Option(args.assembly).foreach(builder.add("--assembly").add(_))
+    Option(args.vepArgs).foreach(builder.add(_))
 
     if (args.addFiles) {
       builder.addFile(args.cachePath)

--- a/core/src/main/scala/org/bdgenomics/cannoli/Vep.scala
+++ b/core/src/main/scala/org/bdgenomics/cannoli/Vep.scala
@@ -39,8 +39,8 @@ class VepArgs extends Args4jBase {
   @Args4jOption(required = false, name = "-executable", usage = "Path to the Ensembl VEP executable. Defaults to vep.")
   var executable: String = "vep"
 
-  @Args4jOption(required = false, name = "-image", usage = "Container image to use. Defaults to quay.io/biocontainers/ensembl-vep:100.2--pl526hecda079_0.")
-  var image: String = "quay.io/biocontainers/ensembl-vep:100.2--pl526hecda079_0"
+  @Args4jOption(required = false, name = "-image", usage = "Container image to use. Defaults to quay.io/biocontainers/ensembl-vep:101.0--pl526hecda079_0.")
+  var image: String = "quay.io/biocontainers/ensembl-vep:101.0--pl526hecda079_0"
 
   @Args4jOption(required = false, name = "-sudo", usage = "Run via sudo.")
   var sudo: Boolean = false

--- a/core/src/main/scala/org/bdgenomics/cannoli/Vep.scala
+++ b/core/src/main/scala/org/bdgenomics/cannoli/Vep.scala
@@ -89,8 +89,6 @@ class Vep(
       .add("--output_file")
       .add("STDOUT")
       .add("--vcf")
-      .add("--vcf_info_field")
-      .add("ANN")
       .add("--no_stats")
       .add("--offline")
       .add("--dir_cache")

